cmd/lncli/cmd_open_channel.go

```diff
@@ -0,0 +1,277 @@
+package main
```

Comment:
This commit (7f96272b84490c7b4507307d6e64a24b1f0b60f6) is showing out of order, ideally it comes before the PSBT one.

Reply:
Yes, it's before. Was just showing incorrectly on GitHub because of the time based sort. Fixed the commit timestamps.